sdcm/nemesis.py

```diff
@@ -2437,11 +2437,17 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         wait_time = random.choice(list_of_timeout_options)
         self.log.debug("Taking down eth1 for %dsec", wait_time)
 
+        if self.target_node.distro.is_rhel_like:
```

Comment:
let's move it it BaseNode level.
Create method `stop_extra_interface()` `start_extra_interface`